### PR TITLE
serve `isIntersecting` polyfill to chromium based browsers

### PR DESCRIPTION
**Pull request by romainmenke**
see : https://github.com/Financial-Times/polyfill-library/issues/1282

### CODE REVIEW

File: polyfills/IntersectionObserver/tests.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/* eslint-env mocha, browser */
+/* global proclaim */
+
+describe('IntersectionObserver', function () {
```
Comment (romainmenke) [inline polyfills/IntersectionObserver/tests.js:4]: A very minimal test to ensure we have at least some indication that the polyfill works.

File: polyfills/IntersectionObserverEntry/tests.js [hunk 2]
```diff
@@ -2,7 +2,38 @@
 /* global proclaim */
 
 describe('IntersectionObserverEntry', function () {
+	var scrollArea;
```
Comment (romainmenke) [inline polyfills/IntersectionObserverEntry/tests.js:5]: Another very minimal test, but focussed on `isIntersecting`

**Review by JakeChampion — APPROVED** [review]

**Comment by romainmenke** [comment]
🤔 browserstack seems to have broken something again.
Going to check again in a few days.

**Comment by romainmenke** [comment]
CI all green

**Pull request merged**